packages/client/src/generation/TSClient/Model.ts

```diff
@@ -418,18 +456,24 @@ ${
     ? `type ${countArgsName}${ifExtensions(
         '<ExtArgs extends runtime.Types.Extensions.Args = runtime.Types.Extensions.DefaultArgs>',
         '',
-      )} = Merge<
+      )} = 
   Omit<${getModelArgName(name, DMMF.ModelAction.findMany)}, 'select' | 'include'> & {
     select?: ${getCountAggregateInputName(name)} | true
   }
->
 `
     : ''
 }
 export interface ${name}Delegate<GlobalRejectSettings extends Prisma.RejectOnNotFound | Prisma.RejectPerOperation | false | undefined${ifExtensions(
       ', ExtArgs extends runtime.Types.Extensions.Args = runtime.Types.Extensions.DefaultArgs',
       '',
     )}> {
+${ifExtensions(
+  `${indent(
+    `[K: symbol]: { types: Prisma.TypeMap<ExtArgs>['model']['${name}'], meta: { name: '${name}' } }`,
```

Comment:
Should we use specific symbol here, rather than index signature for any symbol? Might it cause us any problems if we decide to have an actual symbol on the models some day?

Reply:
Yes, now that it's clear users will need to have a peer dependency of Prisma Client to publish extensions, this can now go away as the same symbols will be shared.